lib/wanda_web/controllers/result_controller.ex

```diff
@@ -4,7 +4,7 @@ defmodule WandaWeb.ResultController do
 
   alias OpenApiSpex.Schema
```

Comment:
Should we still continue calling this `result_controller`? Everything was moved to `Executions` so maybe `executions_controller`. And together with this the view code, and most probably the routes (`/api/checks/executions`)

Reply:
@nelsonkopliku is taking care of this in another PR